ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -23,6 +23,7 @@ SS:
   - failed_alarm
   - lost_power_alarm
   - power_status
+  - bypass_status
```

Comment:
Apologies for the confusion; please remove here and add the `bypass_status` field as an `opt_uses` on the `UPS` in the GENERALTYPES.yaml

Reply:
updated